lib/constants/enums.ts

```diff
@@ -180,3 +180,12 @@ export enum DisconnectionReason {
   AuthFailureInvalidSignature = 12,
   WireProtocolErr = 13,
 }
+
+export enum AlertType {
+  LowBalance = 0,
+}
+
+export enum ChannelBalanceSide {
```

Comment:
I think it might be nice to just call this `ChannelSide` since we'd use the same term when referring to things other than balance and could reuse this enum, plus it's less typing :smile:.